### PR TITLE
geomyidae: init at 0.34

### PR DESCRIPTION
**Pull request by rapenne-s**
###### Motivation for this change

This adds a simple gopher daemon.
I registered myself as maintainer

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rapenne-s** [comment]
I accidentally deleted the branch on my fork which closed the PR.

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -7955,6 +7955,12 @@
     githubId = 2157287;
     name = "sohalt";
   };
+  solene = {
```
Comment (SuperSandro2000) [inline maintainers/maintainer-list.nix:7958]: Please put this into a separate commit with the message ``maintainers: add solene``.

File: pkgs/servers/gopher/geomyidae/default.nix [hunk 2]
```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchgit, libressl }:
+
+stdenv.mkDerivation rec {
+
```
Comment (SuperSandro2000) [inline pkgs/servers/gopher/geomyidae/default.nix:4]:
```suggestion
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 98851` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package failed to build:</summary>
  <ul>
    <li>geomyidae</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
```
builder for '/nix/store/1xka1qyqm5zgirfqb6i0l5mndmwyfysd-geomyidae.drv' failed with exit code 1; last 10 log lines:                                                                            * [new tag]         v0.30      -> v0.30                                                                                                                                                     * [new tag]         v0.31      -> v0.31
   * [new tag]         v0.31.1    -> v0.31.1
   * [new tag]         v0.32      -> v0.32
   * [new tag]         v0.32.1    -> v0.32.1
   * [new tag]         v0.32.2    -> v0.32.2
   * [new tag]         v0.33      -> v0.33
   * [new tag]         v0.34      -> v0.34
  fatal: empty string is not a valid pathspec. please use . instead if you meant to match all paths
  Unable to checkout refs/tags/0.34 from git://bitreich.org/geomyidae.
cannot build derivation '/nix/store/932ncp9i66lna9dr1fd0abr44q4gmvmy-geomyidae-0.34.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/ra3vkksfcnsm83rwzb3pr2y5zmhq09cc-env.drv': 1 dependencies couldn't be built
```
